serverless/aws/lambda/src/main/scala/sttp/tapir/serverless/aws/lambda/model.scala

```diff
@@ -8,7 +10,41 @@ case class AwsRequest(
     body: Option[String],
     isBase64Encoded: Boolean
 )
+
 case class AwsRequestContext(domainName: Option[String], http: AwsHttp)
 case class AwsHttp(method: String, path: String, protocol: String, sourceIp: String, userAgent: String)
 
-case class AwsResponse(cookies: List[String], isBase64Encoded: Boolean, statusCode: Int, headers: Map[String, String], body: String)
+case class AwsResponse(isBase64Encoded: Boolean, statusCode: Int, headers: Map[String, String], body: String)
+
+object AwsResponse {
+  def badRequest(body: String = ""): AwsResponse =
+    AwsResponse(isBase64Encoded = false, StatusCode.BadRequest.code, Map.empty, body)
+}
+
+/**
+  * I could NOT managed to enforce CDK to use payload version 2.0 At time of this writing no valuable
+  * documentation of aws/apigateway2 exists
+  * @see https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-apigatewayv2.PayloadFormatVersion.html
+  */
+case class AwsRequestV1(
```

Comment:
if this is used only for CDK, maybe it should be moved to the CDK module? Alongside the v1 -> v2 translation layer?

Reply:
You are correct that currently, only CDK module use that v1 model. But both request models (v1 and v2) stem directly from AWS Lambda specification. Neither of them constitutes any shape of internal module assumption. According to my intuition, both belong to the same place regardless of their current usage.

Reply:
You might be right, but then the main question to ask here is "what is the responsibility of the `lambda` module". Is it modelling Lambda's data structures? If so, then definitely v1 and v2 belong there.

But I would say that it's rather code, that's needed to run tapir endpoints inside a lambda. In other words, the module's task is to provide some functionality (there are also modules that provide data structures, and that's totally fine). Given such a perspective, I'd say only the data structures needed for that particular functionality should be there - what do you think?

(btw. these kinds of definitions - which module is for what - probably should be in the docs, though it's not there yet)

Reply:
I like your profound insight into responsibilities separation. I could not appreciate that more. I do not feel competent to argue with you on that matter, but I will share my thoughts. From my perspective, the **lambda** module is directly dependent on the AWS environment and concepts. So it is all about how we will define "run lambda" meaning. Suppose running lambda assumes conforming to AWS realms that imply knowledge about both request versions. The ultimate handler has no choice but to work with those models. The particular implementation depends on configuration, which is not always under our control. As a lambda client (developer), I would like to get access to valuable abstractions to get it from there.

Reply:
Please argue ;)

As a developer trying to use tapir + lambda, would you ever use the V1 request model? If it's only usage is for CDK, let's keep it there. I understand the "all things lambda go into the `lambda` module" approach, but I still don't think that's the responsibility of the module. It's not a "common-like" module, but providing some specific functionality

Reply:
I probably misunderstood the intended purpose of the lambda module. My reasoning is simple, v1 and v2 should belong to the same space because they are equivalent. 

This case overlaps with that discussion about LambdaHandler. Both boil down to the same matter. LambdaHandler abstractions support the existence of request version duality -  that is the handler's sole purpose. That assumption also implies equality of both versions.

I will try to grasp the root concepts behind that module better. Maybe, in fact, lambda is not the best inhabitant for such models. But according to my intuition, both requests versions belong to the same place.

Reply:
Well I'll keep my position that the lambda module is not a place where we try to model AWS datastructures, or a "common" module that serves as the shared code repository for other modules, but code that is needed to deploy a lambda using tapir.

But ultimately you're the PR author so it's up to you. If it will bother me at some point I'll simply change it in the future ;)

Reply:
Fair enough, but I would prefer to find common ground. I tend to question stuff because it is my way of learning. I am not going to insist on my solution. It is about striving to get closer to your point and understand your assumptions behind that module.